errors/team-not-existent.md

```diff
@@ -0,0 +1,11 @@
+# No Token Allowed
```

Comment:
Isn't this title bad? It should be `The team doesn't exists` or something like that right?

Reply:
Fixed. Thank you! 😊  0ae0158